ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/Owner.scala

```diff
@@ -3,17 +3,48 @@
 
 package com.daml.ledger.on.memory
 
+import com.daml.ledger.api.v1.experimental_features.{
+  CommandDeduplicationFeatures,
+  CommandDeduplicationPeriodSupport,
+  CommandDeduplicationType,
+  ExperimentalContractIds,
+}
 import com.daml.ledger.runner.common.{Config, ConfigProvider}
 import com.daml.ledger.participant.state.kvutils.app.Runner
 import com.daml.ledger.resources.ResourceOwner
+import com.daml.platform.apiserver.LedgerFeatures
+import com.daml.platform.services.time.TimeProviderType
 
 object Owner {
-  // Utily if you want to spin this up as a library.
-  def apply(config: Config[Unit]): ResourceOwner[Unit] =
+  // Utility if you want to spin this up as a library.
+  def apply(config: Config[Unit]): ResourceOwner[Unit] = {
+    val features = LedgerFeatures(
```

Comment:
For better visibility and readability, you might want to move the definition of this `LedgerFeatures` into a `def reasonableDefaultsForLedgerFeatures` or similar (alternatively, you could make it a constant if you overwrite the `staticTime` field here).